### PR TITLE
Update_freezer

### PR DESCRIPTION
**Pull request by Karthi8883**
updated freezer monitoring points

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -192,6 +192,18 @@ DX2RC:
...
+FRZM:
+  description: "Freezer monitoring."
+  is_abstract: true
+  opt_uses:
+   - failed_alarm
+   - run_status
+   - compressor_run_status
+   - run_time_accumulator
+  uses:
+   - temperature_sensor
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:204]: what temperature?

**Review by tasodorff — CHANGES_REQUESTED** [review]
model these after other chiller types. where is the start command? water temps? is this actually a chiller?

see precedent in the CH.yaml file.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 1 above)
Comment (Karthi8883) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:204]: This is a freezer used in kitchen. Will remove this canonical type from chiller and add it to the CRREF.yaml approved yesterday. The temperature is the freezer temperature.

**Comment by Karthi8883** [comment]
Will create new pull request based on recently approved pull requests.

**Pull request closed without merging**